ui/components/AutomationsTable.tsx

```diff
@@ -122,18 +119,17 @@ function AutomationsTable({ className, automations, hideSource }: Props) {
     },
     {
       label: "Last Updated",
-      value: () => (
-        <Timestamp time={""} />
-      ),
+      value: () => <Timestamp time={""} />,
     },
   ];
 
   if (hideSource) fields = _.filter(fields, (f) => f.label !== "Source");
 
   return (
     <FilterableTable
+      initialSelections={{ "type:Kustomization": true }}
```

Comment:
These prop changes need to happen on every FilterableTable - aka sources and reconciled objects - right now those tables don't crash but also don't change the url

Reply:
Yeah this was just a debug thing. Nice catch. Removed.